pkg/hvs/services/hosttrust/trust_report.go

```diff
@@ -296,36 +298,38 @@ func getHostManifestMap(hostManifest *types.HostManifest, flavorParts []cf.Flavo
 					})
 				}
 				if !reflect.DeepEqual(hostInfo.HardwareFeatures, taModel.HardwareFeatures{}) {
-					if hostInfo.HardwareFeatures.CBNT != nil {
+					pfQueryAttrs = append(pfQueryAttrs, models.FlavorMetaKv{
+						Key:   "hardware.feature.CBNT.enabled",
+						Value: hostInfo.HardwareFeatures.CBNT.Enabled,
+					})
+					if hostInfo.HardwareFeatures.CBNT.Enabled {
```

Comment:
Very Important to mark these places and use supported flag instead of "!= nil" check, better to add todo

Reply:
Added todo comment.